src/styles/components/menu.scss

```diff
@@ -28,3 +32,13 @@
   width: 126px;
   font-size: 0;
 }
+
+.menu-option::after {
```

Comment:
@giovramirez it could be write as:

```
.menu-option {
   color: $grey-light;
   text-decoration: none;
   margin: 0 12px;

   &::after {
     content: ".";
   }
}
```